nbviewer/providers/github/handlers.py

```diff
@@ -281,6 +297,18 @@ def get(self, user, repo, ref, path):
             self.redirect(tree_url)
             return
 
+        return raw_url, blob_url, tree_entry
+
+    @gen.coroutine
+    def deliver_notebook(self, user, repo, ref, path, raw_url, blob_url, tree_entry):
+        """
+        provider_url:
+            URL to the notebook document upstream at the provider (e.g., GitHub)
+        breadcrumbs: list of dict
+            Breadcrumb 'name' and 'url' to render as links at the top of the notebook page
+        executor_url: str, optional
+            URL to execute the notebook document (e.g., Binder)
```

Comment:
`provider_url`, `breadcrumbs` and `executor_url` are not function parameters here. Let's remove them from the docstring and optionally document the actual parameters.